src/zsh_completion/_firejail.in

```diff
@@ -171,6 +171,7 @@ _firejail_args=(
 #ifdef HAVE_APPARMOR
     '--apparmor[enable AppArmor confinement]'
     '--apparmor.print=-[print apparmor status name|pid]:firejail:_all_firejails'
+    '--apparmor=[enable AppArmor confinement with a custom profile]'
```

Comment:
Is `--apprmor <PROFILE>` really supported?

```suggestion
    '--apparmor=-[enable AppArmor confinement with a custom profile]'
```

Reply:
The whitespace syntax is supported only for profile files, not for command line (I didn't know it's needed). I'll try to fix that.